icicle_v3/backend/cpu/src/cpu_device_api.cpp

```diff
@@ -92,3 +92,9 @@ class CpuDeviceAPI : public DeviceAPI
 };
 
 REGISTER_DEVICE_API("CPU", CpuDeviceAPI);
+
+class CpuRefDevice : public CpuDeviceAPI
```

Comment:
I think that if you rebase you can simply do 
REGISTER_DEVICE_API("CPU_REF", CpuDeviceAPI);
without having the class again. I remember that this was solved at some point. Not sure about that but if possible remove CpuRefDevice and use CpuDeviceAPI instead.

Reply:
It doesn't work, so I left it as it was. Do you want to look at it together?

Reply:
not that important. Ok leave it like that then